### ISSUE TITLE
More granular jobs limits

### ISSUE THREAD

**Issue by t-izbassar** [zerocracy/datum#263]
According to [$3](http://datum.zerocracy.com/pages/policy.html#3) I need to have 512 points to get more than 3 jobs, which seems not so reasonable as starting from 256 I can join real projects. Also working on sandbox projects means sometimes waiting more times. So for now for example I can only wait for PR approvals then issue closing until I can start new task. This time I could spend in other task. I'm proposing to have those limits:

- 3 jobs if below 256
- 5 jobs if below 512
- 7 if below 1024
- 9 if below 2048
- 16 if below 4096
- 24 if greater than 4096

What do you think?

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@t-izbassar I'm not sure now. We have to see what will happen with our gang in the next few months. Let's keep this open.

**Comment by g4s8** [comment]
@t-izbassar @yegor256 I agree that it's hard for new users to earn 512 reputation when they can work only on 3 tasks at the same time. Maybe we can change this limit smoothly, I mean by some formula, not by static limits?
